src/security-server/system-test/src/intTest/resources/container-files/etc/xroad/conf.d/acme.yml

```diff
@@ -4,6 +4,11 @@ eab-credentials:
       mac-key-base64-encoded: true
       members:
         "DEV:COM:1234":
-          kid: keyid_1
-          mac-key: k21lms0bWFjXzAxdd39Dnsk2nr20am1WAx
+          #kid: keyid_1
+          #mac-key: Jzy09jeiXT0yrpZ7ZRmOV0esf4Fs1Z02
```

Comment:
is this needed?

Reply:
I removed from the system-tests but left it for dev-stack setup for convenience, so it would be easier to switch between two different approaches to let the acme server know which type of certificate to return